lib/jellyfish/component/hls/manager.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Jellyfish.Component.HLS.Manager do
```

Comment:
We need more logs in this module. At least:
* info that the uploading process has  started
* info that the uploading process has  finished and its status
* info that we remove hls from a disk